zk_toolbox/crates/zk_inception/src/forge_utils.rs

```diff
@@ -12,3 +13,21 @@ pub fn fill_forge_private_key(
     }
     Ok(forge)
 }
+
+pub async fn check_the_balance(forge: &ForgeScript) -> anyhow::Result<()> {
+    let mut not_enough_money = true;
+    while not_enough_money {
+        let (success, address) = forge
+            .check_the_balance(MINIMUM_BALANCE_FOR_WALLET.into())
+            .await?;
+
+        not_enough_money = !success;
+
+        if not_enough_money {
+            if common::PromptConfirm::new(format!("Address {address:?} doesn't have enough money to deploy contracts do you want to continue?")).ask() {
+                break;
+            }
+        }
```

Comment:
I think if `not_enough_money == true` and the user prompts to quit, then we should raise an error from there so that program execution is stopped.

Reply:
but it's not quit.. 
It's yes or no, if no it will check again , if yes it will proceed. 

I think it's common knowledge that you can cancel everything using Ctrl + c